client/plots/genomeBrowser.controls.js

```diff
@@ -96,7 +98,7 @@ class GbControls {
 				}
 			})
 		const div = this.dom.variantFilterHolder.append('div').style('display', 'none')
-		filterInit({
+		this.filterUI.variant = await filterInit({
```

Comment:
the `variant filter` should be stateless as the filter is consisted of VCF INFO fields but not dict terms, thus should not be affected by state change

Reply:
ok, I'll fix. Are there places besides mass app, where a cohort filter + other filter uses `filterInit()`? For example, would `block.mds2.vcf.numericaxis.AFtest.js` break with the change to `getFilterExcludingPill()`? I'm not familiar enough where to look for that usage.